### PR TITLE
Move database-migrate-biome feature into stable

### PR DESCRIPTION
**Pull request by RyanLassigBanks**

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -68,11 +70,7 @@ experimental = [
 circuit = ["reqwest", "serde_json", "splinter/sawtooth-signing-compat", "dirs"]
 circuit-auth-type = ["circuit"]
 circuit-template = ["splinter/circuit-template", "circuit"]
-database-migrate-biome = [
-    "splinter/biome-credentials",
-    "splinter/biome-key-management",
```
Comment (agunde406) [inline cli/Cargo.toml]: This change should be in a separate commit from the stabilization commit, with some explanation in the commit message about this change.

File: cli/Cargo.toml [hunk 2]
```diff
@@ -69,12 +69,8 @@ experimental = [
...
-    "splinter/biome-notifications"
-]
-
+database-migrate-biome = ["splinter/biome"]
+node-alias = ["dirs", "reqwest"]
```
Comment (peterschwarz) [inline cli/Cargo.toml:73]: Looks like an extra addition
